service/src/test/java/tds/session/repositories/impl/ExternalSessionConfigurationRepositoryImplIntegrationTests.java

```diff
@@ -24,7 +24,7 @@ public void shouldFindExternalSessionConfigurationByClientName() {
         Optional<ExternalSessionConfiguration> maybeExternalSessionConfiguration = externalSessionConfigurationRepository.findExternalSessionConfigurationByClientName("SBAC");
         assertThat(maybeExternalSessionConfiguration).isPresent();
         assertThat(maybeExternalSessionConfiguration.get().getClientName()).isEqualTo("SBAC");
-        assertThat(maybeExternalSessionConfiguration.get().getEnvironment()).isEqualTo("SIMULATION");
+        assertThat(maybeExternalSessionConfiguration.get().getEnvironment()).isEqualTo("Development");
```

Comment:
I believe Development is what is usually used?  So that seems correct to me.  Are you thinking it should be SIMULATION in the seed data?